### PR TITLE
Editorial changes

### PR DESCRIPTION
**Pull request by kenchris**
Editorial changes after review with our internal team.

### CODE REVIEW

File: src/content/en/updates/posts/2016/03/web-enabled-internet-of-things.markdown [hunk 1]
```diff
@@ -25,6 +25,12 @@ The Internet of Things is on everyone's lips these days, and
 it makes tinkerers and programmers like me very excited. Nothing
 is cooler than bringing your own inventions to life and
 being able to talk to them!
+
+But IoT devices that install apps that you rarely use on your phone or PC
+can be annoying. We can take advantage of the Physical Web project, progressive
+apps, and Bluetooth&#42; Low Energy using using Web Bluetooth an Node.js on
+the Intel&#42; Edison board to make IoT devices more intuitive and less
+intrusive.
 </p>
```
Comment (jpmedley) [inline src/content/en/updates/posts/2016/03/web-enabled-internet-of-things.markdown]: The second sentence is a bit of a mouth full. Can you spit it, or at least take another shot?

Also on line 31 an >> and

File: src/content/en/updates/posts/2016/03/web-enabled-internet-of-things.markdown [hunk 2]
```diff
@@ -53,10 +59,10 @@ use cases might be better suited for Web NFC or Web USB. For example,
 Web USB is preferred if you require a physical connection for security
 reasons.
 
-The web site can also serve as a Progressive Web Apps (PWA).
+The website can also serve as a Progressive Web Apps (PWA).
```
Comment (jpmedley) [inline src/content/en/updates/posts/2016/03/web-enabled-internet-of-things.markdown]: App, not Apps

File: src/content/en/updates/posts/2016/03/web-enabled-internet-of-things.markdown [hunk 3]
```diff
@@ -53,10 +59,10 @@ use cases might be better suited for Web NFC or Web USB. For example,
...
 We encourage readers to check out
 [Google's explanation](https://developers.google.com/web/progressive-web-apps)
-of PWAs. PWAs are site that have a responsive app like user
+of PWAs. PWAs are sites that have a responsive app like user
```
Comment (jpmedley) [inline src/content/en/updates/posts/2016/03/web-enabled-internet-of-things.markdown]: Need a comma between responsive and app-like. Need a hyphen in app-like.

File: src/content/en/updates/posts/2016/03/web-enabled-internet-of-things.markdown [hunk 4]
```diff
@@ -112,19 +118,20 @@ The `package.json` file for the project looks like this:
 }
 {% endhighlight %}
 
-## Announcing the web site
+## Announcing the website
 
-The latest version (M49) of Chrome on Android supports Physical Web, which
+The latest version (M49) of Chrome&reg; on Android&#42; supports Physical Web, which
```
Comment (jpmedley) [inline src/content/en/updates/posts/2016/03/web-enabled-internet-of-things.markdown]: Change to "Beginning with version 49, Chrome on Android supports..." We're two weeks away from 50 going to stable.

**Comment by jpmedley** [comment]
I gave this an editing pass. There are a few things that need tweaked.

**Comment by PaulKinlan** [comment]
@kenchris can you look at the suggestions and do some fixes, once that happens I will pull it in.

**Comment by kenchris** [comment]
@PaulKinlan just did, hope I got it right. Maybe @jpmedley wants to have a quick look again.

**Pull request merged**